python/drned_xmnr/op/base_op.py

```diff
@@ -158,10 +158,11 @@ def setup_drned_env(self, trans):
                                   'set /drned-xmnr/drned-directory or the ' +
                                   'environment variable DRNED')
         elif drdir == "builtin" or drdir is None:
-            env['DRNED'] = os.path.join(self.dev_test_dir, 'drned')
-            drdir = env['DRNED']
+            drdir = os.path.join(self.dev_test_dir, 'drned')
         else:
-            env['DRNED'] = drdir
+            pass
```

Comment:
I suggest you remove these two lines, block `else: pass` is useless.

Reply:
Sure, pass doesn't do anything. But it also tell  the reader that we didn't just forget the else-branch.

Anyways, I'm not married to it...